controllers/testdata/declarative/case_6/deploy/04_cr.yaml

```diff
@@ -1,37 +0,0 @@
-# Test:
-# When a user/odh provides a odh-trusted-ca-bundle, a configmap is created and mounted to mlmd grpc and apiserver
-# When a user provides a cABundle in the DSPA, it is also included in the concatenated dsp custom ca cert configmap
-# When external db is used the server config created for api server uses tls=true
-# MLMD grpc server mounts the dspa cert and passes it into grpc server
-# When a user provides a caBundleFileMountPath, it will be used to mount the ca bundle
-# When a user provides ca bundle configmapkey, it will be used instead of default one
-# When a user provides a ArtifactSignedURLExpirySeconds, it will be used instead of default
```

Comment:
this test was doing more than just testing podtopodtls, but I don't want to block this pr on re-adding this back in, but we should not forget to re-add this once the pr is merged, as a separate task item

Reply:
I created a Jira for that: https://issues.redhat.com/browse/RHOAIENG-13123